src/Compilers/Test/Resources/Core/CompilerTestResources.csproj

```diff
@@ -346,6 +346,7 @@
     <EmbeddedResource Include="SymbolsTests\snKey2.snk" />
     <EmbeddedResource Include="SymbolsTests\snPublicKey.snk" />
     <EmbeddedResource Include="SymbolsTests\snPublicKey2.snk" />
```

Comment:
Yup, and I also forgot to do the extra validation I had planned. Should be fixed now.